server/tokens.go

```diff
@@ -280,3 +343,13 @@ func (c *Server) tokenRouter() chi.Router {
 	r.Method(http.MethodPost, "/bulk/redemption/", middleware.InstrumentHandler("BulkRedeemTokens", handlers.AppHandler(c.blindedTokenBulkRedeemHandler)))
 	return r
 }
+
+// New end point to generated marked tokens
+func (c *Server) markedTokenRouter() chi.Router {
+	r := chi.NewRouter()
+	if os.Getenv("ENV") == "production" {
+		r.Use(middleware.SimpleTokenAuthorizedOnly)
+	}
+	r.Method(http.MethodPost, "/{type}", middleware.InstrumentHandler("IssueTokens", handlers.AppHandler(c.newBlindedTokenIssuerHandler)))
+	return r
+}
```

Comment:
A router can hold a collection of endpoints, so it's not strictly necessary we create a new router for this endpoint- though there may be reasons to.  We could reuse the existing router.

The way I have done this in the past, is that I've created new routers for API version.  Check out how it's done on reputation https://github.com/brave-intl/bat-reputation/tree/master/controllers.

So if we go with the `v2` name instead of `new`, it makes sense to me to create a new router.  And maybe it makes sense to create a new v2 file as well - but I don't want to force a refactor if that's complicated.

Let me know what you think

Reply:
Yes, I think it makes sense to refactor and have a v2 file instead. Shouldn't be a considerable refactor.

Reply:
Finally kept a second router. Renamed both routers to `tokenRouterV1` and `tokenRouterV2`.